Core/Graphics/InternalInclude/Babylon/Graphics/FrameBuffer.h

```diff
@@ -7,14 +7,14 @@ namespace Babylon::Graphics
 {
     class DeviceContext;
 
-    struct ViewPort
+    struct Rect
     {
         float X{0.0f};
         float Y{0.0f};
         float Width{1.0f};
         float Height{1.0f};
```

Comment:
nit: I would recommend changing the defaults if the name is changing. Must change the defaults below also.
```suggestion
        float X{};
        float Y{};
        float Width{};
        float Height{};
```